cogs/ahk/ahk.py

```diff
@@ -743,7 +743,34 @@ async def solved(self, inter: disnake.AppCmdInter):
 
         await inter.channel.edit(
             archived=True,
-            applied_tags=inter.channel.applied_tags + [solved_tag],
+            applied_tags=inter.channel.applied_tags + ([] if solved_tag in inter.channel.applied_tags else [solved_tag]),
+        )
+        
+    @commands.slash_command(description="Unark your post as solved.", guild_ids=[AHK_GUILD_ID])
+    async def unsolved(self, inter: disnake.AppCmdInter):
+        if (
+            not isinstance(inter.channel, disnake.Thread)
+            or inter.channel.parent.id != HELP_FORUM_CHAN_ID
+        ):
+            raise commands.CommandError("This command should just be run in help channel posts.")
+
+        if inter.author != inter.channel.owner:
+            raise commands.CommandError("Only post author can mark as unsolved.")
+
+        solved_tag = disnake.utils.get(inter.channel.parent.available_tags, name="Solved!")
+        if solved_tag is None:
+            raise commands.CommandError("Solved tag not found")
+
+        try:
+            await inter.send(
+                "The post has been opened and lost the solved tag. The post can be resolved at any time by using /solved again."
+            )
+        except:  # hdafdsjkhfjdas
```

Comment:
blank excepts are bad, in this case you probably want to catch `disnake.HTTPException`